autofit/non_linear/result.py

```diff
@@ -311,7 +311,7 @@ def search_internal(self):
 
         try:
             return self.paths.load_search_internal()
-        except FileNotFoundError:
+        except (FileNotFoundError, RecursionError):

```

Comment:
Oh yuck why is a RecursionError occurring? We should get to the bottom of that really

Reply:
I will remove it and then merge, and post an issue when it crops up again.